Userland/Utilities/tee.cpp

```diff
@@ -26,53 +27,39 @@ static Vector<int> collect_fds(Vector<const char*> paths, bool append, bool* err
     }
 
     Vector<int> fds;
-    for (const char* path : paths) {
-        int fd = open(path, oflag, mode);
-        if (fd < 0) {
-            perror("failed to open file for writing");
-            *err = true;
-        } else {
-            fds.append(fd);
-        }
+    for (auto path : paths) {
+        int fd = TRY(Core::System::open(path, oflag, mode));
+        fds.append(fd);
```

Comment:
Technically we could preallocate fds, to avoid allocation in the loop, while you are on it
```suggestion
    TRY(fds.try_ensure_capacity(paths.size())
    for (auto path : paths) {
        int fd = TRY(Core::System::open(path, oflag, mode));
        fds.unchecked_append(fd);
    }
```

Reply:
Thanks. Done as suggested. Doing `paths.size() + 1` so we account for `STDOUT_FILENO` as well.